debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -728,6 +728,11 @@ public static IntervalHandlingMode parse(String value, String defaultValue) {
      */
     public enum SnapshotMode implements EnumeratedValue {
 
+        /**
+         * Performs a snapshot of data and schema upon each connector start.
+         */
+        ALWAYS("always", true, true, false),
```

Comment:
@vjuranek Thinking about that, should not the snapshot be executed also in case of schema history error? The snapshot should be executed under all condiitons.

Reply:
@jpechane my understanding is that `ALYWAS` should be the same as `INITIAL`, but run on each connector start, therefore I use same as for `INITIAL`. Of course do know if I get it right and @Naros meant it in [DBZ-5626](https://issues.redhat.com/browse/DBZ-5626) in this way. @Naros , any thoughts?

Reply:
Hi @vjuranek I agree with @jpechane here, this ultimately should mirror MySQL's _always_ implementation or any other connector that currently supports _always_.

Reply:
fixed in https://github.com/debezium/debezium/pull/4022